var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -97,10 +98,11 @@ class Qgis(CMakePackage):
     # optionals
     depends_on('postgresql@8:', when='+postgresql')  # for PostGIS support
     depends_on('gsl', when='+georeferencer')  # for georeferencer
-    depends_on('grass@7.0.0', type=('build', 'link', 'run'), when='+grass7')  # for georeferencer
+    # grass@7.8.1 is the first version that supports proj@6
```

Comment:
But this package still supports PROJ 4, right? In that case, 7.8.1 isn't required.

Reply:
No it does not support `proj@4`.  I was not able to get `qgis` to concretize and build with `proj@4`.  That why I went to `proj@6` and updated a zillion packages to try to get them working.

Reply:
I remember building it with `proj@4` up to QGIS@3.8.1 inclusive. LTR version, @3.4 though must have dropped the support at some point.

Reply:
Even in the latest version, it seems QGIS supports proj@4 unless one wants to build with grass@7.

Reply:
> Even in the latest version, it seems QGIS supports proj@4 unless one wants to build with grass@7.

Thanks for that info.  Does it work with any versions of GRASS before 7?  I suppose GRASS is optional, but I believed I needed it...

I think the PROJ.4/6 thing is going to be like Python 2/3: It's giving us a lot of pain right now, but in the next 1-2 years, things will have pretty much converted over and we won't be worrying about it anymore.